ESLint alias 경로(@/) 인식 오류 발생

## 어떤 버그인가요?

ESLint가 @/ 경로를 해석하지 못해 오류가 발생

## 어떤 상황에서 발생한 버그인가요?

```
C:\Users\user\MainProject\oz_externship_frontend\src\utils\index.ts
  1:1   error  Resolve error: typescript with invalid interface loaded as resolver  import/no-unresolved
  1:1   error  Resolve error: typescript with invalid interface loaded as resolver  import/export
  1:15  error  Unable to resolve path to module '@/utils/cn'
```

## 예상 결과

ESLint가 @/ alias 경로를 정상적으로 인식
barrel 파일에서 경로 관련 에러 제거

--- a/eslint.config.js
+++ b/eslint.config.js
@@ -62,7 +62,9 @@ export default tseslint.config(
         version: 'detect', // 설치된 React 버전 자동 감지
       },
       'import/resolver': {
-        typescript: true,
+        typescript: {
+          project: "./tsconfig.app.json",
+        },
         node: true,
       },
     },
@@ -157,6 +159,9 @@ export default tseslint.config(
           },
         },
       ],
+
+      "import/no-unresolved": "error",
+      "import/export": "error",
     },
   },
 

--- a/package.json
+++ b/package.json
@@ -36,6 +36,7 @@
     "clsx": "^2.1.1",
     "eslint": "^9.25.0",
     "eslint-config-prettier": "^10.1.5",
+    "eslint-import-resolver-typescript": "^4.4.4",
     "eslint-plugin-import": "^2.32.0",
     "eslint-plugin-react": "^7.37.5",
     "eslint-plugin-react-hooks": "^5.2.0",

